package.json

```diff
@@ -17,6 +17,8 @@
   },
   "dependencies": {
     "koa": "^2.5.2",
+    "koa-parse": "0.0.1",
```

Comment:
It looks like we don’t need this. It’s not hurting anything, but probably best to remove it at some point: `npm uninstall --save koa-parse`